src/shared/ui/alert/Alert.tsx

```diff
@@ -8,15 +8,15 @@ import { AlertProps } from '.';
 
 import csx from './Alert.scss';
 
-export const Alert = ({ idx, message, type, onClose }: AlertProps) => {
+export const Alert = ({ message, type = 'error', onClose }: AlertProps) => {
   const render = usePortal();
 
   const className = [csx.alert, csx[type]].join(' ');
 
   return render(
     <div className={className}>
```

Comment:
Just a question: would it work with template string? I mean: className={`${csx.alert} ${csx[type]`}? If so, shouldn't we stick to one convention?

Reply:
Y it will